Documentation/core-repos.md

```diff
@@ -18,6 +18,7 @@
 |[dotnet/templating](https://github.com/dotnet/templating)         |Templating engine used by `dotnet new`|
 |[dotnet/winforms](https://github.com/dotnet/winforms)             |Windows Forms|
 |[dotnet/wpf](https://github.com/dotnet/wpf)                       |Windows Presentation Foundation (WPF)|
+|[dotnet/maui](https://github.com/dotnet/maui)                     |.NET Multi-platform App UI, a framework for building native device applications spanning mobile, tablet, and desktop|
```

Comment:
Based on [this comment](https://github.com/dotnet/core/pull/4797#issuecomment-662159390):

```suggestion
|[dotnet/maui](https://github.com/dotnet/maui)                     |.NET Multi-platform App UI|
```